src/ImageSharp.Drawing/Pens/Pen{TPixel}.cs

```diff
@@ -26,16 +27,16 @@ namespace ImageSharp.Drawing.Pens
     public class Pen<TPixel> : IPen<TPixel>
         where TPixel : struct, IPixel<TPixel>
     {
-        private static readonly float[] EmptyPattern = new float[0];
-        private readonly float[] pattern;
+        private static readonly ReadOnlySpan<float> EmptyPattern = new float[0];
+        private readonly ReadOnlySpan<float> pattern;
```

Comment:
Don't think so. `System.Buffers.Primitives` looks very experimental to me.

The `S.B.Buffer<T>` and `S.B.ReadOnlyBuffer<T>` structs are really simple cousins of `Span<T>`, but the other stuff happening here ... really strange. Still haven't figured out how exactly they do refcounting. (It's ways more simple in C++!)

Maybe I'm gonna try to introduce some of them as an ImageSharp internal as I did with `BufferSpan<T>`, but I'm not sure if their design fits our needs. Have to go deeper.